src/Compare.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of composer/semver.
+ *
+ * (c) Composer <https://github.com/composer>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace Composer\Semver;
+
+use Composer\Semver\Constraint\VersionConstraint;
+use InvalidArgumentException;
+
+class Compare
+{
+    /**
+     * Whether $version1 > $version2.
+     *
+     * @param string $version1
+     * @param string $version2
+     *
+     * @return bool
+     */
+    public static function greaterThan($version1, $version2)
```

Comment:
the issue with namespaced functions is that functions cannot be autoloaded, forcing to eager-load the file, which then also cause issues in case the file exists in several locations (global install and in the project, or inside the composer phar and in the project...)